packages/taxdat/tests/testthat/test_read_taxonomy_data_sql.R

```diff
@@ -0,0 +1,68 @@
+username=Sys.getenv("username")
+password=Sys.getenv("password")
+
+library(testthat)
+#test read_taxonomy_data_sql function
+test_that("read_taxonomy_data_sql works",{
+  expect_error(
+    taxonomy_data=read_taxonomy_data_sql(username=NULL,password=password,locations=14)
+    )
+  expect_error(
+    taxonomy_data=read_taxonomy_data_sql(username=username,password=NULL,locations=14)
+    )
+  expect_error(
+    taxonomy_data=read_taxonomy_data_sql(username=username,password=password,locations=NULL)
+    )
+  expect_error(
+    taxonomy_data=read_taxonomy_data_sql(username=username,password=password,locations="AFR::ETH")
+    )
+  expect_error(
```

Comment:
Again, I'm not sure what this test is saying about the data. I'm not sure why you expect this call to fail.  Given that you have all calls that you expect errors for, maybe it's worth pointing out that ```expect_error({...})```, passes iff `...` throws an errorr, while ```expect_error({...}, NA)``` passes if it doesn't throw an error.  Either way, have you run these tests on idmodeling to see if they pass there?

Reply:
This test is saying that location argument can't be a name but a uid (location id). i've only run the tests that don't need to pull data from the server and i though we decided to skip those tests that require pulling data from the server? but will update the code to pass when no error is reported.

Reply:
I see the confusion. If someone did change the code in such a way that pulling locations by name worked, we would be happy (and it shouldn't fail tests).

Reply:
I thought you were going to write the successful sql tests, but just not run them except on idmodeling